modules/ROOT/pages/8.2.0-release-notes.adoc

```diff
@@ -42,6 +42,23 @@ The following premium plugin updates were released alongside {productname} 8.2.0
 
 // For information on the **<Premium plugin name 1>** plugin, see: xref:<plugincode>.adoc[<Premium plugin name 1>].
 
+=== Suggested Edits
+
+The {productname} {release-version} release includes an accompanying release of the **Suggested Edits** premium plugin.
+
+**Suggested Edits** includes the following improvement.
+
+=== Operation descriptions now shows the type of content that was changed.
+// #TINY-12598
+
+Previously, the **Suggested Edits** plugin displayed operation cards without indicating the type of content being modified. As a result, reviewers had to open each suggestion individually to determine whether it affected a paragraph, image, table, or another content type.
+
+This lack of context slowed the review process, particularly in documents with numerous suggestions, as users could not quickly identify or prioritize edits by content type.
+
+In {release-version}, the **Suggested Edits** plugin now displays contextual descriptions on each operation card. The plugin automatically detects the content type involved and shows descriptive labels such as “added paragraph,” “deleted image,” or “modified table.” Supported content types include paragraphs, headings, images, tables, block quotes, code blocks, horizontal rules, inline text, and more.
+
+For information on the **Suggested Edits** plugin, see: xref:suggestededits.adoc[Suggested Edits].
```

Comment:
```suggestion
Previously, the **Suggested Edits** plugin only displayed the word "content" in the suggestion card description, as a generalised context for each suggestion.

In {release-version}, the **Suggested Edits** plugin displays the type of content as well as the type of edit in the card description, to provide more context for each suggestion.

For information on the **Suggested Edits** plugin, see: xref:suggestededits.adoc[Suggested Edits].
```
This is an improvement, so it shouldn't be described as a bug. We also aren't using operation or "modified" to describe suggestions or edits. Edits should refer to the actions made by a user, and suggestions refer to how these edits are displayed in the review.